lib/person.rb

```diff
@@ -24,11 +29,24 @@ def book_rents(rental)
     @books.push(rental)
   end
 
+  def self.create_prompt
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+    if valid_input?(person_type)
+      person_type
+    else
+      puts 'Invalid Input'
+      nil
+    end
+  end
+
+  def self.valid_input?(person_type)
+    [1, 2].include?(person_type.to_i)
+  end
+
```

Comment:
- [ ] You should follow the single responsibility principle. All requests for input and input validation should've been placed in the `main.rb` file. Please [check this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid
) article if you need a refresher.